examples/browser-service-worker/webpack.config.js

```diff
@@ -0,0 +1,51 @@
+'use strict'
+
+const path = require('path')
+const webpack = require('webpack')
+
+module.exports = {
+  mode: "development",
+  devtool: 'source-map',
+  entry: './src/main.js',
+  output: {
+    path: path.join(__dirname, 'dist'),
+    filename: 'main.js',
+    publicPath: '/'
+  },
+  devServer: {
+    serveIndex: true,
+    index: './index.html',
+    compress: true,
+    port: 3000,
```

Comment:
Does this port need aligning with the port in the URLs/screenshots in the README?